src/utils/device.js

```diff
@@ -96,3 +96,30 @@ module.exports.isLandscape = function () {
 module.exports.isIOSOlderThan10 = function (userAgent) {
   return /(iphone|ipod|ipad).*os.(7|8|9)/i.test(userAgent);
 };
+
+/**
+ * Check if running in a browser.
+ */
+// we need to check a node api that isn't mocked on either side
```

Comment:
I think you can just use normal comments throughout for these all properties. We don't actually generate docs from this, it's mostly all developer comments, but in JSDoc form. (with preference to capitalized / ending in period comments)